### PR TITLE
Update Docs on UTM

### PR DESCRIPTION
**Pull request by kurbycchua**
1. Added new UTM tags per this release: https://github.com/mixpanel/mixpanel-js/releases/tag/v2.55.1
2. Added note on UTM persistence

### CODE REVIEW

File: pages/docs/tracking-best-practices/traffic-attribution.mdx [hunk 1]
```diff
@@ -2,19 +2,23 @@
...
 
-- utm_source: track the source where traffic is coming from, including a website or advertiser
-- utm_medium: track the advertising medium, including email and banner ads
-- utm_campaign: track the campaign name associated with the traffic
-- utm_content: track the specific link within in an ad that a user clicked
-- utm_term: track keywords associated with campaigns
+- utm_source: source where traffic is coming from, including a website or advertiser
+- utm_medium: advertising medium, including email and banner ads
+- utm_campaign: campaign name associated with the traffic
+- utm_content: specific link within in an ad that a user clicked
```
Comment (tdumitrescu) [inline pages/docs/tracking-best-practices/traffic-attribution.mdx]:
```suggestion
- utm_content: specific link within an ad that a user clicked
```

File: pages/docs/tracking-best-practices/traffic-attribution.mdx [hunk 2]
```diff
@@ -2,19 +2,23 @@
...
+- utm_term: keywords associated with campaigns
+- utm_id / utm_campaign_id : unique identification number of a campaign
```
Comment (tdumitrescu) [inline pages/docs/tracking-best-practices/traffic-attribution.mdx]:
```suggestion
- utm_id / utm_campaign_id : unique identifier of a campaign
```
(This is still typically a string, isn't it?)

File: pages/docs/tracking-best-practices/traffic-attribution.mdx [hunk 3]
```diff
@@ -2,19 +2,23 @@
...
+- utm_source_platform: marketing platform responsible for directing traffic
+- utm_creative_format: type of ad creative, example: display, native, video, search
```
Comment (tdumitrescu) [inline pages/docs/tracking-best-practices/traffic-attribution.mdx]:
```suggestion
- utm_creative_format: type of ad creative, for example: display, native, video, search
```

File: pages/docs/tracking-methods/sdks/javascript.md [hunk 4]
```diff
@@ -54,12 +54,13 @@ mixpanel.track_pageview({"page": "Pricing"});
...
 
-When UTM parameters for an identified user are seen for the first time, these will also be stored on the user profile as `initial_utm_source`, `initial_utm_campaign`, `initial_utm_medium`, `initial_utm_term`, and `initial_utm_content`.
+When UTM parameters for an identified user are seen for the first time, these will also be stored on the user profile as `initial_utm_source`, `initial_utm_campaign`, `initial_utm_medium`, `initial_utm_term`, `initial_utm_content`, `initial_utm_id`, `initial_utm_source_platform`, `initial_utm_campaign_id`, `initial_utm_creative_format`, and `initial_utm_marketing_tactic`.
 
-In addition to UTM parameters, Mixpanel will also add any advertising click IDs to events fired. These include `dclid`, `fbclid`, `gclid`, `ko_click_id`, `li_fat_id`, `msclkid`, `ttclid`, `twclid`, `wbraid`
+In addition to UTM parameters, Mixpanel will also add any advertising click IDs to events fired. These include `dclid`, `fbclid`, `gclid`, `ko_click_id`, `li_fat_id`, `msclkid`, `ttclid`, `twclid`, `wbraid`, `sccid`.
```
Comment (tdumitrescu) [inline pages/docs/tracking-methods/sdks/javascript.md]:
```suggestion
In addition to UTM parameters, Mixpanel will also add any advertising click IDs to events fired. These include `dclid`, `fbclid`, `gclid`, `ko_click_id`, `li_fat_id`, `msclkid`, `sccid`, `ttclid`, `twclid`, `wbraid`.
```
(Maintain alphabetical order)

File: pages/docs/tracking-methods/sdks/javascript.md [hunk 5]
```diff
@@ -54,12 +54,13 @@ mixpanel.track_pageview({"page": "Pricing"});
...
 
+> Note: UTM parameters are by default persisted as [Super Properties](/docs/tracking-methods/sdks/javascript#super-properties). To disable this behaviour, use the SDK initialization option `{stop_utm_persistence: true}` (refer to out [Release Note](https://github.com/mixpanel/mixpanel-js/releases/tag/v2.52.0) in github).
```
Comment (tdumitrescu) [inline pages/docs/tracking-methods/sdks/javascript.md]:
```suggestion
> Note: UTM parameters are by default persisted as [Super Properties](/docs/tracking-methods/sdks/javascript#super-properties). To disable this behavior, use the SDK initialization option `{stop_utm_persistence: true}` (refer to our [Release Notes](https://github.com/mixpanel/mixpanel-js/releases/tag/v2.52.0) in GitHub).
```

**Review by tdumitrescu — COMMENTED** [review]
Thanks, just a few bits of copy-editing cleanup to do

File: pages/docs/tracking-best-practices/traffic-attribution.mdx (hunk 1 above)
Comment (kurbycchua) [reply pages/docs/tracking-best-practices/traffic-attribution.mdx]: updated

File: pages/docs/tracking-best-practices/traffic-attribution.mdx (hunk 2 above)
Comment (kurbycchua) [reply pages/docs/tracking-best-practices/traffic-attribution.mdx]: updated

File: pages/docs/tracking-best-practices/traffic-attribution.mdx (hunk 3 above)
Comment (kurbycchua) [reply pages/docs/tracking-best-practices/traffic-attribution.mdx]: updated

File: pages/docs/tracking-methods/sdks/javascript.md (hunk 4 above)
Comment (kurbycchua) [reply pages/docs/tracking-methods/sdks/javascript.md]: updated

File: pages/docs/tracking-methods/sdks/javascript.md (hunk 5 above)
Comment (kurbycchua) [reply pages/docs/tracking-methods/sdks/javascript.md]: updated

**Comment by kurbycchua** [comment]
> Thanks, just a few bits of copy-editing cleanup to do

@tdumitrescu thanks for the comments, I have updated the changes suggested, please have look to see if I missed any

**Review by tdumitrescu — APPROVED** [review]
🎉

**Pull request merged**